src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -356,12 +354,19 @@ export function ChangeDataView({
         <TextBasedLanguagesList
           textBasedLanguages={textBasedLanguages}
           selectedOption={triggerLabel}
-          onChange={(lang) => {
+          onChange={async (lang) => {
             setTriggerLabel(lang);
             setPopoverIsOpen(false);
             setIsTextBasedLangSelected(true);
+            let queryString = `SELECT * FROM "${trigger.title}"`;
+            if (currentDataViewId) {
```

Comment:
what will be trigger.title in this case where currentDataViewId is unset ?

Reply:
This is set by the applications that are using the dataview picker. We can add a check here